nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc.yaml

```diff
@@ -0,0 +1,22 @@
+nelmio_api_doc: ~
```

Comment:
What about removing `~` here? I think it's not strictly needed and avoid errors if one only uncomment entries below.